### PR TITLE
Add HyperRogue 11.3a

### PR DESCRIPTION
**Pull request by SeekingMeaning**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Review by ran-dall — CHANGES_REQUESTED** [review]
@SeekingMeaning Have you checked if this needs an `zap` stanza?

**Comment by SeekingMeaning** [comment]
Not 100% sure how I would go about doing that, any pointers?

**Comment by ran-dall** [comment]
@SeekingMeaning You can it manually with an uninstaller app or you can try @nrlquaker's [createzap project](https://github.com/nrlquaker/homebrew-createzap).

File: Casks/hyperrogue.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'hyperrogue' do
+  version '11.3a'
+  sha256 'f6829a1ad8e640c39f99bd164ead9d70e6dc24879cf9c5b4d542ce07f6324ffd'
+
+  url "https://roguetemple.com/z/hyper/hyperrogue-#{version.tr('.', '')}.dmg"
```
Comment (miccal) [inline Casks/hyperrogue.rb:5]: Please change this to:
```
url "https://roguetemple.com/z/hyper/hyperrogue-#{version.no_dots}.dmg"
```

File: Casks/hyperrogue.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  appcast 'https://github.com/zenorogue/hyperrogue/releases.atom'
```
Comment (miccal) [inline Casks/hyperrogue.rb:6]: I am not sure that is is a suitable `appcast` -- it lists version 11.3o as the latest, and seems to only provide the source code.

Comment (ran-dall) [reply Casks/hyperrogue.rb:6]: @SeekingMeaning `https://roguetemple.com/z/hyper/download.php` maybe?

**Comment by vitorgalvao** [comment]
@SeekingMeaning Still interested in this?

**Comment by SeekingMeaning** [comment]
Whoops, closing this for now

**Pull request closed without merging**